src/assets/locales/es.json

```diff
@@ -1,169 +1,169 @@
-  "room-settings.access-private-subtitle": "Solo aquellos con el enlace pueden unirse",
-  "room-settings.access-shared-link-subtitle": "Solo aquellos con el enlace pueden unirse",
-  "room-settings.access-invite": "Solo con invitación",
-  "room-settings.access-invite-subtitle": "Invitar a personas con un enlace que se pueda revocar",
-  "room-settings.revoke-confirm": "¿estás seguro?",
+  "room-settings.access-shared-link": "Vínculo compartido",
+  "room-settings.access-private-subtitle": "Solo aquellos con el vínculo pueden unirse",
+  "room-settings.access-shared-link-subtitle": "Solo aquellos con el vínculo pueden unirse",
+  "room-settings.access-invite": "Invitación exclusiva",
+  "room-settings.access-invite-subtitle": "Invita a personas con un vínculo que puede ser anulado",
+  "room-settings.revoke-confirm": "¿estás seguro(a)?",
   "room-settings.revoke-confirm-yes": "sí",
   "room-settings.revoke-confirm-no": "no",
-  "room-settings.revoke": "revocar",
+  "room-settings.revoke": "anular",
   "room-settings.access-public": "Público",
-  "room-settings.access-public-subtitle": "Listado en la página de inicio",
-  "room-info.title": "Información de la habitación y la escena",
-  "room-info.scene-info": "Información de la escena",
-  "close-room.message": "Si cierras esta sala, tú y los demás saldreis de la sala y la sala se cerrará permanentemente.\n\n¿Estás seguro? Esta acción no se puede deshacer.",
-  "close-room.confirm": "Sí, cerrar habitación",
+  "room-settings.access-public-subtitle": "Listado en la página principal",
+  "room-info.title": "Información sobre la sala y escena",
+  "room-info.scene-info": "Información sobre la escena",
+  "close-room.message": "Al cerrar esta sala, te expulsarás a ti y a otros y la eliminarás permanentemente.\n\n¿Estás seguro(a)? Está acción no puede ser revertida.",
+  "close-room.confirm": "Sí, cerrar la sala",
   "close-room.cancel": "Cancelar",
-  "promote.message": "Promocionar a un usuario otorgará acceso completo a la configuración de la sala y las herramientas de moderación.\n\n¿Estás seguro?",
-  "promote.confirm-prefix": "Sí, Promocionar ",
+  "promote.message": "Al promover al usuario, le darás acceso completo a las características de la sala y a las herramientas de moderación.\n\n¿Estás seguro(a)?",
+  "promote.confirm-prefix": "Sí, promover a ",
   "promote.cancel": "Cancelar",
-  "entry.room": "vestíbulo",
-  "entry.enter-room-title": "Vestíbulo",
-  "entry.enter-room": "Entrar",
-  "entry.leave-room": "Salir de la habitación",
+  "entry.room": "antesala",
+  "entry.enter-room-title": "Antesala",
+  "entry.enter-room": "Entrar a la sala",
+  "entry.leave-room": "Abandonar la sala",
   "entry.change-scene": "Elegir escena",
-  "entry.in-lobby-notice": "Está viendo esta habitación desde el vestíbulo.",
-  "entry.screen-prefix": "Entrar en ",
-  "entry.desktop-screen": "Pantalla",
+  "entry.in-lobby-notice": "Estás mirando esta sala desde la antesala.",
```

Comment:
"Estás mirando esta sala desde la antesala." should be "Estás viendo esta sala desde la antesala."